example/src/test/java/com/graphhopper/example/HeadingExample.java

```diff
@@ -0,0 +1,94 @@
+package com.graphhopper.example;
+
+import com.graphhopper.GHRequest;
+import com.graphhopper.GHResponse;
+import com.graphhopper.GraphHopper;
+import com.graphhopper.config.Profile;
+import com.graphhopper.util.Helper;
+import com.graphhopper.util.Parameters;
+import com.graphhopper.util.shapes.GHPoint;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.io.File;
+import java.util.Arrays;
+
+public class HeadingExample {
```

Comment:
For the other examples (RoutingExample.java etc.) we took a slightly different approach where the examples are in `src/main/java`, and they are called from a separate test class in `src/test/java`. Actually, I am not sure why (@karussell?), but we should probably stick to one approach. Btw `RoutingExample.java` also has an example for heading, so not sure if we need more/both?

Reply:
Ok, from src/main they can copy the entire file. But from src/test it would also be possible to copy the main method. But no real need to change this, so for this PR better move HeadingExample to src/main like the others, if we keep it at all.

Reply:
Using src/main is also nice, because this way the examples are contained in the jar.

Reply:
> Btw `RoutingExample.java` also has an example for heading, so not sure if we need more/both?

In [a18c45f](https://github.com/graphhopper/graphhopper/pull/2714/commits/a18c45fb9181401947a419524194e30def301b50) I allowed myself to delete the heading part in RoutingExample.java.

I updated HeadingExample.java to be in src/main and an HeadingExampleTest.java to ensure that it compiles and assures correctly.